draft-packages/filter-menu-button/KaizenDraft/FilterMenuButton/FilterMenuButton.tsx

```diff
@@ -49,17 +56,23 @@ export interface FilterMenuButtonProps {
 
 export const FilterMenuButton = ({
   id,
+  automationId,
   labelText,
   children,
   metadata,
+  autoHide = "on",
   isDropdownVisible,
   toggleDropdown,
   hideDropdown,
   onFilterClear,
 }: FilterMenuButtonProps) => {
   const dropdownId = `${id}-dropdown`
   return (
-    <div id={id} className={styles.filterMenuButton}>
+    <div
+      id={id}
+      data-automation-id={automationId}
```

Comment:
Is there a reason that I‘d want the test id to be on the containing div and not the control itself? What are these test ids usually used for? I usually prefer getByRole so try to not use the getByTestId function

Reply:
Literally just for testing purposes. It's an old pattern to use `automationId` that we're deprecating (the complex components haven't had the fix yet due to the amount of work needed). Ideally we should avoid using test ids as you do, but it's just there as an escape hatch when you can't target with anything else (which generally signifies another problem, but that's a whole different discussion).

If you don't need to use the test id immediately, I reckon we just not add it.

In terms of why it's on the container, it's just a case of ease of use as drilling down is generally easier than drilling up.

Reply:
Good points,

We are going to rethink how we'd like to use the automated tests before implementing extra props, I removed it from this PR